packages/site/src/components/LinkItem.svelte

```diff
@@ -0,0 +1,28 @@
+<script>
+    export let href=''
+</script>
+
+<li>
+    <a
+      href={href}><slot />
+    </a>
+</li>
+  
+  <style>
```

Comment:
The indentation here looks off-- can you run `npm run format` in the root?

(I'm not sure why the linter didn't complain-- will have to fix that on my end)